lib/Actor/include/Actor/ActorID.h

```diff
@@ -35,6 +35,11 @@ struct ActorID {
   auto operator<=>(ActorID const& other) const = default;
 };
 
+inline std::ostream& operator<<(std::ostream& o, ActorID const& id) {
+  o << id.id;
+  return o;
+}
+
```

Comment:
gtest checks if a type as an operator<< - this was added so that gtest assertions would print nice values instead of hex encoded bytes. But I can certainly improve the output as suggested,